lib/common/resolve.js

```diff
@@ -141,23 +141,27 @@ function fullResolve(modulePath, sourceFile, settings) {
         }
     }
 
-    const configResolvers = settings['import/resolver'] || {
-        node: settings['import/resolve'],
-    }; // backward compatibility
 
-    const resolvers = resolverReducer(configResolvers, new Map());
+    const configResolvers = settings['import/resolver'] ? settings['import/resolver'] : 
+                            settings['import/resolve'] ?  {
+                                node: settings['import/resolve'], // backward compatibility
+                            } : null;
 
-    for (const pair of resolvers) {
-        const name = pair[0];
-        const config = pair[1];
-        const resolver = requireResolver(name, sourceFile);
-        const resolved = withResolver(resolver, config);
+    if(configResolvers) {
 
-        if (!resolved.found) continue;
+        for (const pair of resolvers) {
```

Comment:
I believe you deleted the resolvers variable and it was undefined when I checked.

Reply:
Oops! Thanks for keeping me honest.